pkg/agent/cniserver/interface_configuration_windows.go

```diff
@@ -228,11 +227,8 @@ func attachContainerLink(ep *hcsshim.HNSEndpoint, containerID, sandbox, containe
 	var err error
 	var hcnEp *hcn.HostComputeEndpoint
 	if isDockerContainer(sandbox) {
-		// Docker runtime
-		attached, err = isContainerAttachOnEndpointFunc(ep, containerID)
-		if err != nil {
-			return nil, err
-		}
+		// Support for Docker runtime has been deprecated in Antrea 2.0.
```

Comment:
Since you have a check in server.CmdAdd/CmdDel/CmdCheck to refuse the request with Docker runtime, we shall not hit this condition in the current function.

Reply:
Updated.